pages/image/[id].js

```diff
@@ -126,9 +190,45 @@ const ImagePage = () => {
         objectFit="contain"
         layout="intrinsic"
       />
+      <>
+        {/* Tooltip element shown when InfoIcon is hovered over */}
+        <TooltipContainer
+          onMouseEnter={toggleTooltip}
+          onMouseLeave={toggleTooltip}
+        >
+          <InfoIcon />
+          <TooltipText show={showTooltip}>
+            <strong>Creativity:</strong>
+            <br />
+            Measures the uniqueness and originality of the AI-generated image.
+            <br />
+            <br />
+            <strong>Technical Quality:</strong>
+            <br /> Evaluates the level of detail, clarity, and overall
+            craftsmanship of the image.
+            <br />
+            <br />
+            <strong>Emotional Impact:</strong>
+            <br /> Considers how well the image captures and conveys an
+            emotional tone or atmosphere.
+            <br />
+            <br />
+            <strong>Coherence:</strong>
+            <br /> Determines how well the various elements of the image fit
+            together harmoniously.
+            <br />
+            <br />
+            <strong>Realism:</strong>
+            <br /> Gauges how accurately and convincingly the image represents
+            its subject matter.
+            <br />
```

Comment:
Please use css to create space and no `<br />` tags. You could use a `headline` instead of a `strong` tag and a `p` tag for each part of the rest of your text.

Reply:
In this updated code, I've created a new TooltipTitle component for the headings and a TooltipTextItem component for the text paragraphs. I've also used CSS properties to add spacing between the elements and change the font size. Finally, I've removed the <br> tags and instead used appropriate HTML tags (<h2> for headings and <p> for paragraphs) to structure the content of the tooltip.